src/schema.ts

```diff
@@ -259,6 +259,7 @@ export const typeDefs = gql`
     grepCodes: [String]
     competenceGoals: [CompetenceGoal]
     coreElements: [CoreElement]
+    crossSubjectTopics(subjectId: String, filterIds: String): [Topic]
```

Comment:
For meg høres det litt sirkulært å hente topics her i og med at en Topic inneheld en Article. Kanskje bedre å lage en custom-type som inneheld kun det du treng for å lage lenkene: Navn på tverrfaglig tema og path til emnet.